pipelines/deploy-fbc-operator/0.1/deploy-fbc-operator.yaml

```diff
@@ -55,6 +55,14 @@ spec:
     - description: Full OCI artifact reference in a format "quay.io/org/repo:tag"
       name: OCI_REF
       type: string
+    - description: Name of the Kubernetes Secret that contains the access token for a private GitHub or GitLab repository
+      name: REPO_TOKEN
+      default: ""
+      type: string
+    - description: Key within the Secret's "data" field that holds the GitHub/GitLab access token
+      name: REPO_KEY
+      default: ""
```

Comment:
just curious, will having an empty string as default cause issues when accessing public repos?

Reply:
For public repositories, we assume that REPO_TOKEN and REPO_KEY are empty strings. In that case, the script will skip to [this line](https://github.com/konflux-ci/tekton-integration-catalog/blob/99f17e5fe44a83e4c216c9a32f87adb203eaec49/pipelines/deploy-fbc-operator/0.1/deploy-fbc-operator.yaml#L207), which does not use a token.

Reply:
right, thanks! sorry I missed that